etc/profile-m-z/megaglest.profile

```diff
@@ -19,7 +21,8 @@ include disable-xdg.inc
 mkdir ${HOME}/.megaglest
 whitelist ${HOME}/.megaglest
 whitelist /usr/share/megaglest
-whitelist /usr/share/games/megaglest	# Debian version
+# Debian version
```

Comment:
As already suggested, let's just drop this comment.

Reply:
I am against this since I am on a distribution that is not debian and the comment helps understand why this path is whitelisted

Reply:
Alrighty, point taken.